packages/astro/src/@types/astro.ts

```diff
@@ -536,6 +536,41 @@ export interface AstroUserConfig {
 	 */
 	redirects?: Record<string, RedirectConfig>;
 
+	/**
+	 * @docs
+	 * @name prefetch
+	 * @type {boolean | PrefetchOptions}
+	 * @description
+	 * Enable prefetching for your site. A prefetch script is automatically added to every page
+	 * in the project. Just add the `data-astro-prefetch` attribute to any `<a />` links on your page
+	 * and you're ready to go!
+	 *
+	 * If you use [View Transitions](https://docs.astro.build/en/guides/view-transitions/), prefetch will
+	 * be enabled automatically with a default of `{ prefetchAll: true }`, which enables prefetching for all links.
+	 * You can configure this option to override the defaults, and even set `false` to disable prefetching
+	 * completely when using View Transitions.
+	 */
+
+	/**
+	 * @docs
+	 * @name prefetch.prefetchAll
+	 * @type {boolean}
+	 * @description
+	 * Enable prefetching for all links by default, similar to all `<a />` links having the `data-astro-prefetch`
+	 * attribute set. You can disable prefetching individually by setting `data-astro-prefetch="false"` on a link.
+	 */
```

Comment:
Are these blocks intended? They are the same as the ones of `PrefetchOptions`

Reply:
Yeah it's intentionally duplicated as I think that's how the docs will scan it. The other properties are also duplicated.